cmd/status-cli/message.go

```diff
@@ -41,7 +43,30 @@ func (cli *StatusCLI) sendContactRequestAcceptance(ctx context.Context, msgID st
 	return nil
 }
 
-func (cli *StatusCLI) sendDirectMessage(ctx context.Context, text string) error {
+func (cli *StatusCLI) randomFailure() func() {
+	nBig, err := rand.Int(rand.Reader, big.NewInt(100))
+	if err != nil {
+		cli.logger.Error("failed to generate random number", "err", err)
+		return nil
+	}
+	n := nBig.Int64()
+	if n >= 40 {
+		return nil
+	}
+
+	cli.backend.StatusNode().WakuV2Service().SkipPublishToTopic(true)
+
+	return func() {
+		cli.backend.StatusNode().WakuV2Service().SkipPublishToTopic(false)
+	}
+}
+
+func (cli *StatusCLI) sendDirectMessage(ctx context.Context, text string, options ...bool) error {
+	randomFailure := false
+	if len(options) > 0 {
+		randomFailure = options[0]
```

Comment:
Would it make more sense to have some options struct than relying on the order of variable arguments? I guess this is just a note, maybe we can create an issue or just "remember" and refactor later to avoid blocking this

Reply:
If we add another option, then yes absolutely, but I think it should be ok for now